src/config.c

```diff
@@ -123,7 +123,7 @@ void parse_config (GromitData *data)
 
   GromitPaintType type;
   GdkRGBA *fg_color=NULL;
-  guint width, arrowsize, minwidth;
+  guint width, arrowsize, minwidth, maxsize;
```

Comment:
Please also call the variable `maxwidth` here. Just to be in line with the `minwidth` before.